source/tests/CMakeLists.txt

```diff
@@ -251,15 +251,19 @@ add_timemory_google_test(type_trait_tests
 add_timemory_google_test(warning_tests
     SOURCES         warning_tests.cpp
     LINK_LIBRARIES  test-werror-flags
-                    test-debug-flags
                     timemory::timemory-core)
+if(NOT WIN32)                    
+    target_link_libraries(warning_tests test-debug-flags)
+endif()
```

Comment:
```suggestion
```

Reply:
There is a hook in `add_timemory_google_test` that will not create the target if `TIMEMORY_BUILD_GOOGLE_TEST` is OFF

Reply:
No, that is not what this is for. See next comment below.